### PR TITLE
Update Go packages

### PR DESCRIPTION
**Pull request by dhaiducek**
ref: https://issues.redhat.com/browse/ACM-11664

### CODE REVIEW

**Review by gparvin — DISMISSED** [review]

**Comment by dhaiducek** [comment]
I started seeing this on the deployment:
```
pods "gatekeeper-operator-controller-7bd846b94c-89j2k" is forbidden:
violates PodSecurity "restricted:v1.24": seccompProfile (pod or containers "kube-rbac-proxy",
"manager" must set securityContext.seccompProfile.type to "RuntimeDefault" or
"Localhost")
```
Adding it back to see whether it resolved it (it was removed in https://github.com/gatekeeper/gatekeeper-operator/pull/256)

**Comment by dhaiducek** [comment]
@mprahl What's the effect of adding `seccompProfile: runtimeDefault` back? I saw it was removed in https://github.com/gatekeeper/gatekeeper-operator/pull/256, but locally I'm seeing the operator pods get killed (I think because of memory and it getting preempted by the Gatekeeper pods) and then it's not able to come back because of the restricted profile.

**Comment by dhaiducek** [comment]
(Alternatively we could patch the Deployment in the tests like we do for the Openshift ones. That might make more sense rather than adding it back...)

**Comment by mprahl** [comment]
> @mprahl What's the effect of adding `seccompProfile: runtimeDefault` back? I saw it was removed in [gatekeeper#256](https://github.com/gatekeeper/gatekeeper-operator/pull/256), but locally I'm seeing the operator pods get killed (I think because of memory and it getting preempted by the Gatekeeper pods) and then it's not able to come back because of the restricted profile.

I think now that we don't need to support OCP 4.10, we can add it back.

**Review by yiraeChristineKim — APPROVED** [review]

File: test/e2e/util/util.go [hunk 1]
```diff
@@ -165,9 +166,18 @@ func Kubectl(args ...string) {
...
-	//nolint:forbidigo
-	fmt.Println(string(output))
+	kubectlCmd := exec.Command("kubectl", args...)
+
+	output, err := kubectlCmd.CombinedOutput()
+	if err != nil {
+		// Reformat error to include kubectl command and stderr output
+		err = fmt.Errorf(
+			"error running command '%s':\n %s: %s",
+			strings.Join(kubectlCmd.Args, " "),
```
Comment (yiraeChristineKim) [inline test/e2e/util/util.go:176]: love it Thanks

File: test/e2e/e2e_suite_test.go [hunk 2]
```diff
@@ -121,27 +119,27 @@ var _ = BeforeSuite(func() {
...
 })
 
-var _ = AfterSuite(func() {
+var _ = AfterSuite(func(ctx SpecContext) {
 	By("tearing down the test environment")
 
 	if affinityNode != nil {
 		if affinityPod != nil {
-			err := K8sClient.Delete(ctx, affinityPod, client.PropagationPolicy(v1.DeletePropagationForeground))
+			err := K8sClient.Delete(ctx, affinityPod, client.PropagationPolicy(metav1.DeletePropagationForeground))
```
Comment (yiraeChristineKim) [inline test/e2e/e2e_suite_test.go:127]: good catch!!

**Pull request merged**